tofu/rancher/custom_cluster/README.md

```diff
@@ -0,0 +1,41 @@
+# Downstream Rancher Cluster rke2/k3s module
+
+This module deploys a downstream cluster on your rancher setup
+
+## Prerequisites
+
+* An api_key from your rancher setup
+* tofu installed on your client machine
+
+## Usage
+
+
+1.  **Create the downstream cluster**
+    * see the [variables section](#sample) to configure the cluster
```

Comment:
This doesn't seem to point anywhere

Reply:
I think it's because the heading below's name is `` Sample `vars.tfvars` `` so just the sample part doesn't point directly to it

Reply:
I meant to fix this before 😂